www/source/demo/windows/steps/2.html.slim

```diff
@@ -0,0 +1,57 @@
+section
+ h1 Create Your Plan
+
+ p At the center of Habitat packaging is the plan. This is a directory comprised of Powershell scripts and optional configuration files that define how you download, configure, make, install, and manage the lifecycle of the software in the package. When writing a plan, it's important to understand that you are defining both how the package is built and how the Habitat service will behave when the Supervisor starts and manages the child process in the package.
+
+ p To create your plan, navigate to the root of your project repo and run the following:
+ 
+ = code(:shell) do
+   |
+     PS C:\contosouniversity> hab plan init --windows
+
+ p Using <code>hab plan init</code> will result in a new directory located at <code>/path/to/repo/habitat</code> on your workstation. A <code>plan.ps1</code> file will be created and the <code>$pkg_name</code> variable in it will be set to the name of the parent directory. Also, any environment variables that you have previouly set (such as <code>HAB_ORIGIN</code>) will be used to populate the respective <code>$pkg_*</code> variables. Because you specified the <code>--windows</code> argument, your plan is a <code>.ps1</code> file with Powershell syntax, if you had not used the <code>--windows</code> argument, a plan.sh bash file would have been generated.
+ p See #{link_to 'hab plan init', '/docs/reference/habitat-cli#hab-plan-init'} for more information on how to use this subcommand.
```

Comment:
This should be `/docs/habitat-cli/#hab-plan-init` (the anchor is being dropped because of a client-side redirect of /docs/reference).